### DISCUSSION TITLE
LADB shows that can't install

### DISCUSSION THREAD

**Discussion by gary0522** [tribalfs/GalaxyMaxHzPub#138] [Q&A] [answered]
![Screenshot_20231220_015552_LADB](https://github.com/tribalfs/GalaxyMaxHzPub/assets/154364570/8dc43d39-fbf4-4342-85f4-187629bcbaf8)

Model: Galaxy S23
Version: Android 14/one UI6.0
Issue: I had try to follow the step to install the apk through the LADB. But it shows that the apk "failed to open" and "permission denied".

**Comment by tribalfs** [comment, marked as answer]
You are using "non-adb shell" mode in ladb. Should be "adb shell" instead. "Pair and shell" be turned on in LADB setting and don't skip pairing.

**Reply by musowm** [reply]
Ladb apk kerak

**Comment by pichulec** [comment]
I recommend shizuku instead of ladb, much better and updates can be done from the app.
